### PR TITLE
Add tests for TransactionHandler.getEntityId()

### PR DESCRIPTION
**Pull request by apeksharma**
**Detailed description**:
Signed-off-by: Apekshit Sharma <apekshit.sharma@hedera.com>

**Which issue(s) this PR fixes**:
Part of #571 . Needed to do #560 in right way, hence P1.

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/transactionhandler/ConsensusCreateTopicTransactionHandlerTest.java [hunk 1]
```diff
@@ -20,6 +20,18 @@
...
+
 class ConsensusCreateTopicTransactionHandlerTest extends AbstractTransactionHandlerTest {
+    @Override
+    protected TransactionHandler getTransactionHandler() {
+        return new ConsensusCreateTopicTransactionHandler();
+    }
 
+    @Override
+    protected Map<String, Integer> getEntityIdFields() {
+        return Map.of("record.receipt.topicID", EntityTypeEnum.TOPIC.getId());
```
Comment (Nana-EC) [inline hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/transactionhandler/ConsensusCreateTopicTransactionHandlerTest.java:35]: the string should be a constant.
Alternatively you can have "record.receipt" as the const and a method that take in the additional part and the method adds "topicID"

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/transactionhandler/ConsensusDeleteTopicTransactionHandlerTest.java [hunk 2]
```diff
@@ -20,6 +20,18 @@
...
+
 class ConsensusDeleteTopicTransactionHandlerTest extends AbstractTransactionHandlerTest {
+    @Override
+    protected TransactionHandler getTransactionHandler() {
+        return new ConsensusDeleteTopicTransactionHandler();
+    }
 
+    @Override
+    protected Map<String, Integer> getEntityIdFields() {
+        return Map.of("body.consensusDeleteTopic.topicID", EntityTypeEnum.TOPIC.getId());
```
Comment (Nana-EC) [inline hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/transactionhandler/ConsensusDeleteTopicTransactionHandlerTest.java:35]: Same thing with body.x.y. Would make the tests cleaner and easy to modify with a single method that returns the key for use here

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/transactionhandler/ConsensusCreateTopicTransactionHandlerTest.java (hunk 1 above)
Comment (apeksharma) [reply hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/transactionhandler/ConsensusCreateTopicTransactionHandlerTest.java:35]: will do the latter in the followup.

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/transactionhandler/ConsensusDeleteTopicTransactionHandlerTest.java (hunk 2 above)
Comment (apeksharma) [reply hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/transactionhandler/ConsensusDeleteTopicTransactionHandlerTest.java:35]: same.

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/transactionhandler/AbstractTransactionHandlerTest.java [hunk 3]
```diff
@@ -20,6 +20,114 @@
...
+    @Test
+    void testGetEntityId() {
+        // given
+        Message.Builder transactionBodyBuilder = TransactionBody.newBuilder();
+        Message.Builder transactionRecordBuilder = TransactionRecord.newBuilder();
+        Map<String, Integer> entityIdFields = getEntityIdFields();
+        for (var entityIdField : entityIdFields.entrySet()) {
+            Iterator<String> fields = Arrays.stream(Strings.split(entityIdField.getKey(), '.')).iterator();
+            String fieldName = fields.next();
+            if (fieldName.equals("body")) {
```
[the remark this answers is not in the document]
Comment (apeksharma) [reply hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/transactionhandler/AbstractTransactionHandlerTest.java:79]: Yeah it is indeed a bit complex.
The thing i don't like about common setups like those are, tests become coupled in weird and unexpected ways.
Could have made reflection logic much simpler, but having plain java logic is more maintainable as a team.
So i'll go with your suggestion.

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/transactionhandler/ConsensusCreateTopicTransactionHandlerTest.java (hunk 1 above)
Comment (apeksharma) [reply hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/transactionhandler/ConsensusCreateTopicTransactionHandlerTest.java:35]: function got removed.

File: hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/transactionhandler/ConsensusDeleteTopicTransactionHandlerTest.java (hunk 2 above)
Comment (apeksharma) [reply hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/transactionhandler/ConsensusDeleteTopicTransactionHandlerTest.java:35]: function got removed.

**Pull request merged**